src/misc.js

```diff
@@ -46,12 +46,8 @@ const Scoring = {
   LEADER: 7
 };
 
-var Action = {
```

Comment:
why?

Reply:
It allows you to turn Action[2] into DISCARD (in addition to turning Action.DISCARD into 2).